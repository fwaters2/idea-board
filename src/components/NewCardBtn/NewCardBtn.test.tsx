import React from "react";
import { NewCardBtn } from ".";
import { render } from "@testing-library/react";

describe("NewCardBtn component", () => {
  const props = {
    handleAddCard: jest.fn(),
  };

  test("should call handleAddCard function when button is clicked", () => {
    const { getByRole } = render(<NewCardBtn {...props} />);
    getByRole("button").click();
    expect(props.handleAddCard).toHaveBeenCalled();
  });
});
