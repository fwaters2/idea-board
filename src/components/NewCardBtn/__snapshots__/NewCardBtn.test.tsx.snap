// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewCardBtn component should render correctly 1`] = `
<div>
  <button
    class="new_card_btn"
  >
    +
  </button>
</div>
`;
